modules/pymol/experimenting.py

```diff
@@ -133,18 +133,54 @@ def minimize(sele='', iter=500, grad=0.01, interval=50, _setup=1, _self=cmd):
             print(" minimize: missing parameters, can't continue")
 
 
-    def dump(fnam,obj,_self=cmd):
+    def dump(fnam, obj, state=0, quiet=1, _self=cmd):
```

Comment:
PyMOL's state handling is 1-based in the Python API, and 0-based in the C-API. So it should be `state=1` here...